redaxo/src/addons/debug/lib/debug.php

```diff
@@ -10,19 +10,6 @@ class rex_debug
     /** @var \Clockwork\Support\Vanilla\Clockwork */
     private static $instance;
```

Comment:
brauchen wir die `rex_debug`-klasse überhaupt noch?
Wäre `rex_debug_clockwork::getInstance` nicht passender als `rex_debug::getInstance`?
Es liefert ja eine Clockwork-Instanz..

Reply:
habe nochmal über die Aufteilung nachgedacht.
Das einzig losgelöste von Clockwork wäre die `getTrace()` mit `$ignoreClasses`.
Die könnte man in eine `rex_debug` lassen und alles andere ist ja Clockwork bezogen und könnte in die `rex_debug_clockwork`

Reply:
Letztlich ist es mir egal :D
Mach mal, wie du denkst (sind ja internal Klassen, daher ist die Struktur nicht so ganz wichtig, und theoretisch jederzeit änderbar).

Reply:
mach ich :kissing_heart: